pkg/networkmanager/mtu.jsx

```diff
@@ -38,7 +38,7 @@ export const MtuDialog = ({ connection, dev, settings }) => {
     const [mtu, setMtu] = useState(settings.ethernet.mtu ? settings.ethernet.mtu : '');
 
     const onSubmit = (ev) => {
-        const mtuNew = parseInt(mtu, 10);
+        const mtuNew = mode == 'auto' ? 0 : parseInt(mtu, 10);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19156-20230801-065114-15e72e1d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
That means that we don't have a test which exercises that very thing (setting MTU to "auto"). That would be nice to add before we land this.

Reply:
Sorry, I merged this before I noticed your review comment. @subhoghoshX mind adding a test?

Reply:
Oops. @jelly no issues, adding it to my todo, thanks for mentioning.

Question: now I wonder, is there a way to run the coverage bot locally on a commit or something?